packages/video_player/tizen/src/video_player_tizen_plugin.cc

```diff
@@ -237,6 +237,7 @@ void VideoPlayerTizenPlugin::SeekTo(
                          [result]() -> void { result(std::nullopt); });
   } catch (const VideoPlayerError &error) {
     result(FlutterError(error.code(), error.message()));
+    return;
```

Comment:
i have a question.
SeekTo() ends after catch(). Is there a reason why `return;` is necessary? How is this related to crash?

Reply:
Sorry, I misunderstand the meaning of try....catch. 
I have added try...catch in onCompleted callback for pause method.